client/packages/system/src/Name/ListView/ImportProperties/ImportPanel.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC, PropsWithChildren } from 'react';
```

Comment:
Maybe this component could be shared as generic with other similar import via csv features?

Reply:
Which now includes this, asset custom properties, and catalogue item custom properties

Reply:
Yeah that's a great idea! There is a lot of boilerplate code here... perhaps a common `ImportModal` component that handles all the tabs etc, and we pass in the example csv generator, import parser...?

Reply:
I reckon let's make a refactor issue for it though, rather than bloat this PR :)